src/pages/Company/components/CompanyInfo.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import styled, { ThemeProvider } from 'styled-components';
+import GlobalStyle from '../../../styles/GlobalStyle';
+import theme from '../../../styles/theme';
+import Card from './Card';
+
+function CompanyInfo({ companyData }) {
+  const { jobListings, companyDesc, companyPhoto } = companyData;
+  let joinedDesc = companyDesc.join(' ');
```

Comment:
```suggestion
  const joinedDesc = companyDesc.join(' ');
```
let은 반드시 재할당이 필요한 경우에만 활용해주세요!